docker/README.md

```diff
@@ -0,0 +1,15 @@
+# Build in Docker
+
+We provide a Dockerfile to build the project in a container.
+
+## Build the container
+
+```bash
+docker build -t rapids-ml:latest -f Dockerfile .
+```
+Please check the [Dockerfile](./Dockerfile) for more configurable build arguments.
+
+## Build the project in the container
+
```

Comment:
it might be nice to show the command to enter the docker image even if its just exec bash

Reply:
Added.